rfc-138-enable-brotli-compression.md

```diff
@@ -0,0 +1,88 @@
+# Enable Brotli compression on GOV.UK
+
+## Summary
+
+Brotli compression is a new compression algorithm that is available in modern browsers that can offer 10-20% better compression over gzip compression. By enabling this option on the CDN users with browsers that support it will receive smaller static files over the wire (HTML, CSS, JavaScript), and those that don't will continue to receive assets that are gzipped.
+
+## Problem
+
+On GOV.UK we should always be looking to reduce the number of bytes a user has to download to view our pages. This is both good for web performance and data consumption on user devices. Any browser that supports WOFF2 fonts will also support Brotli compression (since this is the compression used in the fonts). According to the latest stats from [caniuse.com](https://caniuse.com/brotli), the Brotli `Accept-Encoding`/`Content-Encoding` functionality is supported by 95% of user browsers globally, and I expect this figure will actually be huger is we only consider UK based users:
```

Comment:
```suggestion
On GOV.UK we should always be looking to reduce the number of bytes a user has to download to view our pages. This is both good for web performance and data consumption on user devices. Any browser that supports WOFF2 fonts will also support Brotli compression (since this is the compression used in the fonts). According to the latest stats from [caniuse.com](https://caniuse.com/brotli), the Brotli `Accept-Encoding`/`Content-Encoding` functionality is supported by 95% of user browsers globally, and I expect this figure will actually be bigger if we only consider UK based users:
```